paths.tex

```diff
@@ -77,9 +77,9 @@ \section{Preliminary Definitions}
 directed temporal networks. To go from undirected to directed, for every edge
 $(e_l,e_r) \in E^2$, add edges $e_l \in E$ and $e_r \in E$ to the directed
 network. To go from directed to undirected, if for every $u,v$ pair in the
-directed network, the number of edges of the form $(u,v,t_0,t_f,w)$ is the same
-as the number of edges of the form $(v,u,t_0,t_f,w)$, then pair up these edges;
-otherwise there is no transformation.
+directed network, the number of edges of the form $e = (u,v)$ is the same
```

Comment:
*Contextual* as in it applies to certain directed graphs depending on how they were generated. For instance, our current DBLP databases graph has all edges duplicated so this definition applies, but for other graph sets/data-representations it becomes incorrect. Either way, since we both think it is not very helpful, we might want to consider deleting it from the document.